proto/v1/database_service.proto

```diff
@@ -1040,7 +1040,11 @@ message ChangeHistory {
   string version = 11;
 
   string description = 12;
+  // The statement is used for preview purpose.
   string statement = 13;
+  // The name of the sheet resource.
+  // Format: projects/{project}/sheets/{sheet}
+  string sheet = 20;
```

Comment:
maybe statement_sheet, as we are expecting schema_sheet and prev_schema_sheet.

Reply:
or sheet -> statement, statement->statement_preview

Reply:
Done.

Keeping "statement" for now as it may require frontend changes.